rcl/test/rcl/test_subscription.cpp

```diff
@@ -937,7 +939,19 @@ TEST_F(
   }
 
   if (is_cft_support) {
```

Comment:
I think that test cases are aware of if that rmw CFT is enabled or rcl CFT is enabled to meet the expectation?

Reply:
Yeah, this checkment should be removed because the rcl CFT should be worked even if the rmw CFT failed.